README.md

```diff
@@ -20,18 +20,176 @@ gem 'kitchen-pester'
+
+* `register_repository` - array (of maps), default is `[]`  
+Allows you to register PSRepositories to download modules from. Useful when you want to use a private feed.  
+This expects a map for each repository to register, the values will be splatted to `Register-PSRepository` (or `Set-PSRepository` if it already exists).
+
+* `install_modules` - array (of maps), default is `[]`  
+Array of maps, that will be splatted to the Install-Module parameters.
+Useful for installing dependencies from a gallery.  
+  ```yaml
+    install_modules:
+      - Name: MyModule
+        Repository: MyPrivateRepo
+        SkipPublisherCheck: true
+  ```
+
+* `copy_folders` - array, default is `[]`  
+Folders (relative to the current directory or absolute) to copy to the System Under test (SUT). The SUT's `$Env:PSModulePath` will be changed for for the session to prepend the parent folder.
+If you are testing a PowerShell module you have built as part of your build process, this enables you to copy this modules.
+  ```yaml
+  verifier:
+    name: pester
+    copy_folders:
+      - output/MyModule
+    pester_install:
+      MaximumVersion: '4.99.999'
+  ```
+
+* `use_local_pester_module` - bool default is `false`  
+Skip installing pester and just use what's available on the box, or what you have copied with the `copy_folders` options.
+
+
+* `pester_install` - map, default is
+  ```ruby
+  {
+    SkipPublisherCheck: true,
+    Force: true,
+    ErrorAction: "Stop",
+  }
+  ```
+  Specify parameters for installing Pester before running the tests. The map will be splatted to the `Install-Module -Name Pester` command.
+  You can use this to install from a private gallery for instance.
+
+* `sudo` - bool, default is `true`. (non-windows only)
+execute all PowerShell calls as sudo. This is useful in some cases, such as when `pwsh` is installed via `snap` and is only available via `sudo` unless you customise the system's configuration.
+
+* `downloads`- map[string, string], defaults to `["./PesterTestResults.xml"] => "./testresults"`.   
+Files to download from SUT to local system, used to download the pester results locally. The key is the remote file, value the local file it should be saved as.
+
+---
+
+## Examples
+
+### Default Windows 2019 Install
+
+If you're testing on a default image of Windows Server 2019, you probably need to replace the builtin Pester module (v3.4.0), and replace the builtin PackageManagement and PowerShellGet to a more recent for the install to work.
+
+Assuming your tests are written for Peter v4, here's a sample configuration:
+```yaml
+verifier:
+  name: pester
+  pester_install:
+    MaximumVersion: '4.99.999'
+  bootstrap: # installs modules from nuget feed by download and unzip.
+        repository_url: "https://www.powershellgallery.com/api/v2"
+        modules:
+          - PackageManagement
+          - PowerShellGet
```

Comment:
nope, it implies it's still under the `verifier:` key, but omits other elements.

Reply:
Hm? `repository_url` and `modules` are three additional indentation levels from `bootstrap` instead of one. Is that intentional? What am I missing? 😄

Reply:
oooops, I thought you where referencing another line... fixing now... sorry.